### DISCUSSION TITLE
Reviewers and editors - update thread

### DISCUSSION THREAD

**Discussion by harisood** [the-turing-way/the-turing-way#2633] [Governance]
This thread keeps you up to date with the reviewers & editors working group as part of the TTW Core Team - reach out if you want to get involved!

# Reviewers & editors - [INSERT DATE] - [INSERT HERDER]

## Useful links

Slack channel: #reviewers-editors-wg in the [Turing Way Slack](https://join.slack.com/t/theturingway/shared_invite/zt-1grobsv2o-Qgu7s1ODpP7oCiqFfdLILw)

## Members
* @dingaaling 
* @vhellon 
* @harisood 
* @EstherPlomp 

## Focus area
To review book content, chapter suggestions and content discussion, and facilitate content creation

## Strategy and long-term work
*Ongoing discussion points for long-term direction and consideration*

## Current focus areas
*What we are working on right now*

## Weekly task list
*Tasks to action this week*

**Comment by harisood** [comment]
This thread keeps you up to date with the reviewers & editors working group as part of the TTW Core Team - reach out if you want to get involved!

# Reviewers & editors - 12/9/22 - Herder: @harisood 

## Members
* @dingaaling 
* @vhellon 
* @harisood 

## Focus area
To review book content, chapter suggestions and content discussion, and facilitate content creation

## Strategy and long-term work
*Ongoing discussion points for long-term direction and consideration*
* Content creation streams - just book content or e.g. videos, learning modules etc?
* Content direction - more structure to creation or keep it free-form?
* Strategy for facilitating review of content created

## Current focus areas
*What we are working on right now*
* Understanding what issues relate to creation and following them up
* Creating structure around effective R&E collaboration
* Creating way to monitor issues in repo (current and future), e.g. tagging, templates


## Weekly task list
*Tasks to action this week*
* [x] Create discussion thread to monitor work - @harisood 
* [ ] Determine schedule/roles/docs etc. required for collaboration - @harisood , @dingaaling , @vhellon 
* [x] Create [HackMD](https://hackmd.io/a-yJw59ZSEKd1G941kuq0g) to track current issue status - @harisood 
* [ ] Determine how to break up issues to review - @harisood 
* [ ] Review issues and categorise on HackMD - @harisood, @dingaaling , @vhellon 
* [x] Create Slack channel on TTW Workspace for reviewers/editors

**Reply by harisood** [reply]
HackMD for reviewing issues: https://hackmd.io/a-yJw59ZSEKd1G941kuq0g

**Reply by harisood** [reply]
Maybe additional task 'start a (private?) channel on TTW slack'. Thoughts? @dingaaling @vhellon

**Reply by dingaaling** [reply]
Thank you for creating all this, @harisood! I like the idea of creating a new channel (e.g. `reviewers-editors-wg`) for our team. That way we don't spam people in the existing `github-related-posts` channel. However, thoughts on making in public so anyone can join if they like?

**Reply by vhellon** [reply]
Thanks so much @harisood!! Thinking for long term work we could perhaps add 'Strategy for facilitating review of content created e.g do we create a bank of reviewers with different expertise'?

**Reply by harisood** [reply]
@vhellon love it will add above!

**Comment by dingaaling** [comment]
# Reviewers & editors - 21 Sept - Herder: @dingaaling 

## Useful links

Slack channel: #reviewers-editors-wg in the [Turing Way Slack](https://join.slack.com/t/theturingway/shared_invite/zt-1grobsv2o-Qgu7s1ODpP7oCiqFfdLILw)

## Members
* @dingaaling 
* @vhellon 
* @harisood 

## Focus area
To review book content, chapter suggestions and content discussion, and facilitate content creation

## Strategy and long-term work
*Ongoing discussion points for long-term direction and consideration*

## Current focus areas
*What we are working on right now*
- Going through older issues
- Setting up team infrastructure

## Weekly task list
*Tasks to action this week*
- [x] Set up internal Slack reminder for this week's herder + to share progress on TTW Slack
- [ ] Go through issues and add to the [HackMD](https://hackmd.io/a-yJw59ZSEKd1G941kuq0g?edit)
- [x] Review comment template @harisood @vhellon (below)

Hello from September 2022 @[USERNAME]]! Hope all is well with you, many months later. The TTW Core team is following up on issues in our backlog and wanted to check if we now should close this issue? If so, feel free to comment or 👍 this comment. If not, please leave a comment on how we could revive this thread and how we can support this work. 

We'll leave the issue open for now, and if we don't hear from you by the end of October we'll mark it as closed - but we are always happy to reopen it! Thank you!

**Reply by dingaaling** [reply]
@harisood + @vhellon I've started adding notes in the HackMD and wanted to ask your feedback on this comment template to revive old threads - essentially to ask the creator whether not it's appropriate to close. As an example: this [issue](https://github.com/alan-turing-institute/the-turing-way/issues/1706). 

Comment template:
Hello from September 2022 @[USERNAME]]! Hope all is well with you, many months later. The TTW Core team is following up on issues in our backlog and wanted to check with you whether it's appropriate to now close this issue? If so, feel free to comment or 👍 this comment. If not, please leave a comment on how we could revive this thread and how we can support this work. Thank you!

**Reply by harisood** [reply]
@dingaaling Love it! Wonder if instead of appropriate we can just say 'check if we now should'?

And then is there something around 'We'll leave the issue open for now, and if we don't hear from you within [INSERT TIME e.g. 2 weeks] we'll mark it as closed - but we are always happy to reopen it!'

**Reply by dingaaling** [reply]
thanks for the feedback @harisood! i've added an update with your comments in the discussion update.

**Reply by dingaaling** [reply]
from co-working call:
- add label to ones we comment/in-review
- excel sheet (shared openly on the slack) for highlighting posts (created/last active in 2019) that we add the "shall we close? comment to
- twitter post + public material about this decision/process so others can weigh in
- blog post could be another outlet for some of this issue content/conversations

**Comment by harisood** [comment]
# Reviewers & editors - 26 Sept - Herder: @harisood 

## Useful links

Slack channel: #reviewers-editors-wg in the [Turing Way Slack](https://join.slack.com/t/theturingway/shared_invite/zt-1grobsv2o-Qgu7s1ODpP7oCiqFfdLILw)
[Google drive](https://drive.google.com/drive/folders/1nM93S515EKsVqcOfF3kxAjLHTDwQcj3T)

## Members
* @dingaaling 
* @vhellon 
* @harisood 
* @EstherPlomp 

## Focus area
To review book content, chapter suggestions and content discussion, and facilitate content creation

## Strategy and long-term work
*Ongoing discussion points for long-term direction and consideration*

## Current focus areas
*What we are working on right now*
- Going through older issues
- Setting up team infrastructure

## Weekly task list
Bit of an update, in order to make sure we're doing this correctly we're going to trial a process on issues that were a) opened in 2019 and b) last active in 2019. The process is to 
1. Review the issues independently to determine whether we should comment saying we'll close it soon if no progress, or just doing a general review
2. Action the issues as appropriate

*Tasks to action this week*
- [x] Create spreadsheet to track relevant issues - [here](https://docs.google.com/spreadsheets/d/1JhcKvwUjNLpLS5uExDGYFUXDeLsl5f746mVmj_gIf0M/edit#gid=0)
- [ ] Confirm wording for general comment
- [ ] Add all issues that fit the criteria above
- [ ] Add independent reviews for issues

##### Comment templates
* To close *
Hello from September 2022 @[USERNAME]]! Hope all is well with you, many months later. The TTW Core team is following up on issues in our backlog and wanted to check if we now should close this issue? If so, feel free to comment or 👍 this comment. If not, please leave a comment on how we could revive this thread and how we can support this work. 

We'll leave the issue open for now, and if we don't hear from you by the end of October we'll mark it as closed - but we are always happy to reopen it! Thank you!

* General review *
*  Hello from September 2022 @[USERNAME]]! Hope all is well with you, many months later. The TTW Core team is currently reviewing issues in our backlog to determine how they can best be taken forward. There is nothing for you to action now, we just wanted to let you know our processes! If you have any thoughts on how to continue moving this issue forwards, please comment below ✨ Thank you!

**Reply by harisood** [reply]
@dingaaling @EstherPlomp @vhellon are we also happy with wording for general review comment?

Also wondering if we should make the Google Drive 'view access' for anyone, and then we are all collaborators who can edit? Atm anyone with link can edit which is maybe not ideal!!

**Reply by EstherPlomp** [reply]
I'm happy with the wording! Should probably have edit access to the four of us and then indeed view or suggestion access to anyone else? 

And thanks for starting the spreadsheet @harisood!

**Reply by dingaaling** [reply]
looks great @harisood! Might suggest some slight wording changes to:

> There is nothing for you to action now, we just wanted to let you know our processes! If you have any thoughts on how to continue moving this issue forwards, please comment below

Instead, how about:

> There is no action needed from you right now. We just wanted to let you know and ask if you had any ideas to share in a comment below for how to move this issue forward?

**Comment by vhellon** [comment]
Just started doing some reviewing. I think for a few issues the work has been done i.e the content is now in the book in some form so we might need a new comment template for those ones. 

Perhaps something like: 
Hello from September 2022 @[USERNAME]]! Hope all is well with you, many months later. The TTW Core team is currently reviewing issues in our backlog to determine how they can best be taken forward. This issue looks like it has now been incorporated in the Turing Way [link to pull request or book content]. Therefore we will now be closing this issue. If you have any questions or concerns, please comment below ✨ Thank you!

**Reply by harisood** [reply]
Wording for issues to close straight away too:

Hello from September 2022 @[USERNAME]]! Hope all is well with you, many months later. The TTW Core team is currently reviewing issues in our backlog to determine how they can best be taken forward. It looks like this issue is no longer being actioned, so we'll be closing it for now - however, please feel free to re-open it to continue working on it at any point! If you have any questions or concerns, please comment below ✨ Thank you!

**Reply by dingaaling** [reply]
Suggestion to combine the comments into one so we don't need to keep changing things:

Hello from 2022 @[USERNAME]! Hope all is well with you, many months later. The TTW Core team is currently reviewing issues in our backlog to determine how they can best be taken forward. This issue looks like it has now been incorporated in the Turing Way in some way or is no longer being actioned. Therefore we will now be closing this issue. If you have any questions or concerns, please comment below ✨ Thank you!

**Comment by dingaaling** [comment]
# Reviewers & editors - 31 Oct - Herder: @dingaaling 

## Useful links

Slack channel: #reviewers-editors-wg in the [Turing Way Slack](https://join.slack.com/t/theturingway/shared_invite/zt-1grobsv2o-Qgu7s1ODpP7oCiqFfdLILw)
[Google drive](https://drive.google.com/drive/folders/1nM93S515EKsVqcOfF3kxAjLHTDwQcj3T)

## Members
* @dingaaling 
* @vhellon 
* @harisood 
* @EstherPlomp 

## Focus area
To review book content, chapter suggestions and content discussion, and facilitate content creation

## Strategy and long-term work
*Ongoing discussion points for long-term direction and consideration*

## Current focus areas
*What we are working on right now*
- Going through older issues to close & comment (focus of JD work on 31/10 co-working session)

## Weekly task list
Bit of an update, in order to make sure we're doing this correctly we're going to trial a process on issues that were a) opened in 2019 and b) last active in 2019. The process is to 
1. Review the issues independently to determine whether we should comment saying we'll close it soon if no progress, or just doing a general review
2. Action the issues as appropriate

*Tasks to action this week*
- [ ] Close out issues with no response to comment/clear closes
- [ ] Add comments to those missing it

##### Comment templates


* To close - new suggestion to reduce changing things in the text *
Hello from 2022 @[USERNAME]! Hope all is well with you, many months later. The TTW Core team is currently reviewing issues in our backlog to determine how they can best be taken forward. This issue looks like it has now been incorporated in the Turing Way in some way or is no longer being actioned. Therefore we will now be closing this issue. If you have any questions or concerns, please comment below ✨ Thank you!

* General review - new suggestion to reduce changing things in the text *
Hello from 2022 @[USERNAME]]! Hope all is well with you, many months later. The TTW Core team is following up on issues in our backlog and wanted to check if we now should close this issue? If so, feel free to comment or 👍 this comment. If not, please leave a comment on how we could revive this thread and how we can support this work.

We'll leave the issue open for now, and if we don't hear from you in a few weeks we'll mark it as closed - but we are always happy to reopen it! Thank you!

**Comment by harisood** [comment]
# Reviewers & editors - 28 Nov - Herder: @harisood 

## Useful links

Slack channel: #reviewers-editors-wg in the [Turing Way Slack](https://join.slack.com/t/theturingway/shared_invite/zt-1grobsv2o-Qgu7s1ODpP7oCiqFfdLILw)
[Google drive](https://drive.google.com/drive/folders/1nM93S515EKsVqcOfF3kxAjLHTDwQcj3T)

## Members
* @dingaaling 
* @vhellon 
* @harisood 
* @EstherPlomp 
* @aleesteele 

## Focus area
To review book content, chapter suggestions and content discussion, and facilitate content creation

## Strategy and long-term work
*Ongoing discussion points for long-term direction and consideration*

## Current focus areas
*What we are working on right now*
- Going through older issues to close & comment. 

## Weekly task list
The latest discussion between @aleesteele, @EstherPlomp & @harisood today: 
- We're now at the stage of reviewing issues people had thoughts on (in Column H of [this spreadsheet](https://docs.google.com/spreadsheets/d/1JhcKvwUjNLpLS5uExDGYFUXDeLsl5f746mVmj_gIf0M/edit#gid=0). We're in danger of thinking 'oh this is cool and should stay open' and continually allowing stale issues to live. The process we decided on is to create a new 'catch-all' issue. If after discussion a specific issue doesn't fit into one of our previous categories of 'can be closed' or 'still live', whoever is there (min 2 people) can decide whether a) this is a core piece of work for the community, in which case it is automatically added to the catch-all issue, or b) it probably isn't a core piece of work for the community, in which case we'll comment saying we're closing the issue, but if the author wants to add it to the catch-all issue they can.

*Tasks to action this week*
- [ ] Close out issues with no response to comment/clear closes
- [ ] Review issues we left notes for in column H

##### New templates

###### Catch-all issue template

Hi there! Welcome to the Reviewers & Editors curated issue list! In this list you'll find a bunch of closed issues that haven't been actioned for a long time and were therefore closed, but we think are a great piece of work for somebody to take the community forward! Please feel free to browse and see if any takes your interest, and then either a) re-open the issue and assign yourself, or b) start a new issue referencing the closed issue with `#<ISSUE_NUMBER>`

We've split them by category to make browsing that little bit easier

## Content
Issues to do with writing content for the book

## Maintenance
Issues to help maintain the Turing Way resource, GitHub, community spaces or more

## Development
Code-focused issues

etc.

###### Template comment for issue a) above

Hello from 2022 @[USERNAME]! Hope all is well with you, many months later. The TTW Core team is currently reviewing issues in our backlog to determine how they can best be taken forward. Although this issue hasn't been actioned for a long time, we think it'd be a great piece of work for somebody else in the community to pick up! For now, we're going to close this issue but reference it in this [catch-all issue] for community members to browse, and possible re-open and take on! If you have any questions or concerns, please let us know, and always feel free to re-open the issue ✨ Thank you!

###### Template comment for issue b) above
Hello from 2022 @[USERNAME]! Hope all is well with you, many months later. The TTW Core team is currently reviewing issues in our backlog to determine how they can best be taken forward. As this issue hasn't been actioned for a long time, we're going to close it for now to make the repo easier to navigate. However, if you think it'd be a great piece of work for somebody else in the community to pick up, please feel free to reference it in this [catch-all issue] for community members to browse, and possible re-open and take on! If you have any questions or concerns, please  let us know, and always feel free to re-open the issue✨ Thank you!

**Reply by harisood** [reply]
@dingaaling @EstherPlomp @vhellon @aleesteele thoughts? What we discussed was:
- Although this is a bit more "we're deciding what stays open", it will help to clean up the repo and create a list of 'core team issues', that we've discussed and think would be great additions to the community. It is also still very community driven, as anyone can still re-open issues/add their issues to the catch-all if they want to!

**Comment by harisood** [comment]
# Reviewers & editors - 16 Jan - Herder: @harisood 

## Useful links

Slack channel: #reviewers-editors-wg in the [Turing Way Slack](https://join.slack.com/t/theturingway/shared_invite/zt-1grobsv2o-Qgu7s1ODpP7oCiqFfdLILw)
[Google drive](https://drive.google.com/drive/folders/1nM93S515EKsVqcOfF3kxAjLHTDwQcj3T)

## Members
* @dingaaling 
* @vhellon 
* @harisood 
* @EstherPlomp 
* @aleesteele 

## Focus area
To review book content, chapter suggestions and content discussion, and facilitate content creation

## Strategy and long-term work
*Ongoing discussion points for long-term direction and consideration*

## Current focus areas
*What we are working on right now*
- Refocus on how we address open issues

## Weekly task list
The latest discussion between @EstherPlomp & @harisood today: 
[Spreadsheet](https://docs.google.com/spreadsheets/d/1JhcKvwUjNLpLS5uExDGYFUXDeLsl5f746mVmj_gIf0M/edit#gid=305769941)
A new system for reviewing issues:
- Four possible categories - 'g' for can be closed immediately, 'r' for keep open, 'c' for core-issue, 'n' for non-core issue
- If 'g', close with message in B2 in Message templates on spreadsheet above
- If 'r', keep it open and add it to #2936 
- If 'c' close and add to #2936 
- If 'n', close with message in B2 in Message templates on spreadsheet above
- If there are comments in col H, at least two people to discuss and then decide on category above.
- We now only require two matching categories to take action (e.g. two 'r' for keep open, not three). The priority is 'g', 'r', 'c', 'n' (so if there are two 'g's and two 'n's, we take action on the 'g's)



*Tasks to action this week*
- [ ] Start using new system, potentially move into a new year?

@dingaaling @EstherPlomp @aleesteele @vhellon thoughts?

**Reply by dingaaling** [reply]
Thanks @harisood ! A few quick thoughts from me too:

- Most of the issues left in the sheet require more complex action (e.g. start a conversation with the core team or create new issue). Do we think that's within our group's remit?
- Should we start on a new set of issues (e.g. 2020) to test the new proposed coding system? 
- I'd advocate for moving towards a system that doesn't require a google sheet

**Reply by harisood** [reply]
@dingaaling Sharing reply here from Slack: 
* We tried with these four options to reduce down the number of actions we could take in fear of getting towards decision fatigue (e.g. removing the reference to it already in the book, and the ambiguity of amber). I feel like any with start a convo or create new issue could be ‘c’ and put in the catch all!
* probs! 2018 or 2020?
* happy to move but not sure what other options will help us keep track!

**Comment by harisood** [comment]
# Reviewers & editors - 1 Feb - Herder: @harisood 

## Useful links

Slack channel: #reviewers-editors-wg in the [Turing Way Slack](https://join.slack.com/t/theturingway/shared_invite/zt-1grobsv2o-Qgu7s1ODpP7oCiqFfdLILw)
[Google drive](https://drive.google.com/drive/folders/1nM93S515EKsVqcOfF3kxAjLHTDwQcj3T)

## Members
* @dingaaling 
* @vhellon 
* @harisood 
* @EstherPlomp 
* @aleesteele 

## Focus area
To review book content, chapter suggestions and content discussion, and facilitate content creation

## Strategy and long-term work
*Ongoing discussion points for long-term direction and consideration*

## Current focus areas
*What we are working on right now*
- Refocus on how we address open issues

## Weekly task list
The new strategy: 
[Spreadsheet](https://docs.google.com/spreadsheets/d/1JhcKvwUjNLpLS5uExDGYFUXDeLsl5f746mVmj_gIf0M/edit#gid=305769941)
A new system for reviewing issues:
- Review issues into a certain category based on the examples in the `Criteria` spreadsheet. The categories are close, add issue to catch-all (#2936) and close it, or leave it open
- Once category is decided, add relevant message from `Message templates`
- If uncertain about an issue, add a link and relevant comments to `Issues for discussion` and the team will discuss when next together

*Tasks to action this week*
- [ ] Finalise old issues with old system in `Tracker`. Probs to do together synchronously
- [ ] Start with this system for 2020 issues

**Comment by dingaaling** [comment]
# Reviewers & editors - 10 March 

## Useful links
- [TTW Core Team Meeting Notes](https://hackmd.io/RIy0un9IS6qimwy0BBWpGA?edit)

## Feedback from Core Team Meeting
- KW: This is great - do we have a chapter describing the process?
- Folks are interested in a training session during Collaboration Cafe
- VH: I will say the open discussion thread does capture a lot of that (for now!) but obvs can work on a chapter
- KW: The chapter could be a list of pros and cons for why this process was decided upon
- BS: There are so many differences between TTW and OSS development, that I would be very interested in the empirically formalise those differences. (let alone, that even for open source, many of the ways we do things are for “historical reasons”, rather than trying multiple ways of doing it and choosing the best). So part of me is sceptical that it won’t work, as this is not the way any other project is doing things, but in the meantime is also intrigued that it may 🙂
